core/templates/dev/head/pages/exploration_player/ExplorationPlayerStateService.js

```diff
@@ -28,14 +28,23 @@ oppia.factory('ExplorationPlayerStateService', [function() {
     },
 
     isInteractionInline: function(stateName) {
+      if (stateName === null) {
+        return true;
+      }
       return exploration.isInteractionInline(stateName);
     },
 
     getInteractionInstructions: function(stateName) {
+      if (stateName === null) {
+        return false;
```

Comment:
I think you mean null.

Reply:
Based on your previous comment, removed such checks from this file and prevented these functions from being called.